babel/localedata.py

```diff
@@ -14,14 +14,18 @@
 
 import os
 import threading
-from itertools import chain
 
 from babel._compat import pickle, string_types, abc
 
 
 _cache = {}
 _cache_lock = threading.RLock()
 _dirname = os.path.join(os.path.dirname(__file__), 'locale-data')
+_locale_identifiers = set(stem for stem, extension in [
```

Comment:
I'm a little hesitant about this happening at import time rather than the first time we need the data...